src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Tables/DeltaTable.cs

```diff
@@ -220,6 +256,16 @@ internal DeltaTable(JvmObjectReference jvmObject)
         public DataFrame History() =>
             new DataFrame((JvmObjectReference)_jvmObject.Invoke("history"));
 
+        /// <summary>
+        /// Generate a manifest for the given Delta Table.
+        /// </summary>
+        /// <param name="mode">Specifies the mode for the generation of the manifest.
+        /// The valid modes are as follows (not case sensitive):
+        /// - "symlink_manifest_format" : This will generate manifests in symlink format
+        /// for Presto and Athena read support.
+        /// See the online documentation for more information.</param>
```

Comment:
Can you provide a link?

Reply:
This is the exact description I take from Delta API. I also have this doubt when I am writing the description here. Is that better to provide a link or follow exact of what they have?

Reply:
Ok, let's keep it as it is.